kong/db/declarative/init.lua

```diff
@@ -802,6 +802,38 @@ function declarative.load_into_cache_with_events(entities, meta, hash)
     return nil, "failed to persist cache page number inside shdict: " .. err
   end
 
+  local worker_count = ngx.worker.count()
```

Comment:
This data doesn't change until reloads, so we can cache it at module level above.